bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/dto/WebPrmInfo.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.linky.internal.dto;
+
+/**
+ * The {@link UserInfo} holds informations about energy delivery point
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+
+public class WebPrmInfo {
+    public class Adresse {
+        public Object adresseLigneUn;
+        public String adresseLigneDeux;
```

Comment:
Is this mix of `Object` and `String` intended ?

Reply:
No, there is no good reason. Align all field to String